modules/serverlist.js

```diff
@@ -0,0 +1,81 @@
+// Made by epsilon#3892
+// https://forum.thd.vg/members/16800/
+
+var config = require("../config.json");
+var request = require("request")
+
+module.exports = function(client) {
+	var module = {};
+
+	var channel, message;
+	channel = client.channels.get(config.server_list.channel);
+	if (!channel) return;
+	channel.fetchMessage(config.server_list.message).then(msg => {
+		message = msg;
+		loop();
+	}).catch(err => {
+		return; // Don't do anything if the message doesn't exist
+	});
+
+	function loop() {
+		request({
+		    url: 'https://api.kag2d.com/v1/game/thd/kag/servers?filters=[{"field":"current","op":"eq","value":"true"},{"field":"connectable","op":"eq","value":true},{"field":"currentPlayers","op":"gt","value":"0"}]',
+		    json: true
+		}, function (error, response, body) {
```

Comment:
change this into an arrow function for consistency

Reply:
im using it to fetch the message (line 11)